app/services/slack/messengers/feedback.rb

```diff
@@ -52,12 +57,10 @@ def call
       def user_detail
         return "*Anonymous report:" unless user
 
-        username = user.username
-        url = "#{ApplicationConfig['APP_PROTOCOL']}#{ApplicationConfig['APP_DOMAIN']}/#{username}"
         format(
           USER_DETAIL_TEMPLATE,
-          username: username,
-          url: url,
+          username: user.username,
+          url: App.url("/#{user.username}"),
```

Comment:
so much better :D

Reply:
@rhymes There are a few things where we have special logic for building URLs that won't just work by overriding `to_param`, so maybe `App.article_url` and `App.user_url` could be sensible additions to the app module.

Reply:
Makes sense as we use them continously! I'll work on them :)